index.html

```diff
@@ -463,6 +463,107 @@ <h3 class="recent-work-article-header">Multi-post-story</h3>
           </article>
         </div>
       </section>
+
+      <section class="contact-me-section">
+        <h3 class="contact-me-header">CONTACT US</h3>
+
+        <p class="contact-me-para">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it Text Input text here See Project
+        </p>
+
+        <form
+          action="https://formspree.io/f/meqvrpkj"
+          method="POST"
+          class="display-flex form flex-d-col"
+        >
+          <input
+            type="text"
+            name="name"
+            id="name"
+            placeholder="Text input"
+            minlength="1"
```

Comment:
The `minlength` attribute here is redundant as we already have a required attribute